src/micrate/cli.cr

```diff
@@ -43,6 +43,7 @@ module Micrate
         raise "Migration name required"
       end
 
+      sleep 1.millisecond
```

Comment:
As @robacarp said on gitter, this isn't enough to fix the issue, we need to check file first

Already working on that :+1: